config/default/testRunner.conf.php

```diff
@@ -389,6 +389,15 @@
              */
             'contentNavigatorType' => 'default'
         ],
+        /**
+         * The plugin renders review panel for test taker
+         * - `default`: the regular layout
+         * - `fizzy`: simplified layout aligned with NextGen styling
+         * @type string
+         */
+        'review' => [
+            'reviewLayout' => 'default'
+        ]
```

Comment:
Should be removed or commented?

Reply:
Nope. This is new option. Unfortunately we do not have place to document new option, like `README.md`. As i see all properties are described in config files, so i did the same - added this "default" option with description.

Reply:
Sorry, my mistake - didn't notice you changed property value